gameswap-service/pom.xml

```diff
@@ -91,6 +91,12 @@
             <version>${dropwizard.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.6.1</version>
+            <scope>test</scope>
```

Comment:
dropwizard supplies jackson, any reason for gson? (i prefer gson too, but if it's bundled with dropwizard we should use it)